cip 0050 (#39)
global-synchronizer-foundation/cips: cip-0050/cip-0050.md

## CIP 0050

<pre>
  CIP: CIP 0050
  Title: Controlled Validator Growth Through Trusted Operators
  Author: Eric Saraniecki 
  Status: Draft

>   Type: Governance

Reviewer: After reflection, the best way to reconcile the view of the Tokenomics committee that it can make these process changes without explicit approval would be to state that this is a change in process and not governance. So definitely let's propose this as a Process CIP, not a Governance CIP.

  Created: 2025-03-11
  Approved: 2025-03-03
  License: CC1
</pre>

## Abstract

- Allow existing Validator operators to create up to 10 additional Validators for third-party participants.
- Redirect new individual Validator applicants to approved existing operators.

> - Maintain a controlled, trusted, and orderly network expansion.

Reviewer: This third point seems to belong under "motivation"

Author: Moved, thanks!

> - Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants.
> - Third-party applicants wishing to run a Validator will be redirected to an existing operator.

Reviewer: ```suggestion
- Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants.
- Third-party applicants wishing to run a Validator will be redirected to an existing operator.
-Whitelisting requests, if the node will not share an existing IP address, will come from the existing Validator operator who sponsors the new node
-Credit for onboarding the Validator goes to the sponsoring Validator Operator
-The Validator onboarding secret will be issued by a Super Validator, as before.
```

> - Pre-approved operators, such as Kiln, will retain their prior allocations (e.g., 30 Validators).

Reviewer: Pre-approved operators, such as Kiln, Dfns and Copper, who have already been granted the right to operate up to 30 validator nodes, will retain that right.

> - Pre-approved operators, such as Kiln, will retain their prior allocations (e.g., 30 Validators).

Author: ```suggestion
- Pre-approved operators, such as Kiln, Dfns and Copper, who have already been granted the right to operate up to 30 validator nodes, will retain that right.
```

> - Operators such as IEU, MPCH, Proof Group, and others already running their own Validators will be given discretion in onboarding third-party Validators.

Reviewer 2: This sounds like we are introducing Tiers of node operators, given the differences in the number of additional Validators certain node operators can run. An idea is to introduce "featured node operators" who can onboard additional Validators at their discretion with no limit. Featured status can be granted after a proven history of running validators reliably.

> - Operators such as IEU, MPCH, Proof Group, and others already running their own Validators will be given discretion in onboarding third-party Validators.

Reviewer: Good point.

> - Operators such as IEU, MPCH, Proof Group, and others already running their own Validators will be given discretion in onboarding third-party Validators.

Reviewer: Eric Saraniecki clarified in cip-discuss that under this process, Validator operators who host ten nodes under this process and want to host more can request additional node hosting permissions from the Tokenomics committee.

- This policy ensures a structured and scalable approach to Validator expansion while maintaining network integrity and supportability.

> - The [application process](https://sync.global/validator-request/) is to become your own operator.

Reviewer: The [application process](https://sync.global/validator-request/) will remain in place for requests to become a Validator Operator.

> - The [application process](https://sync.global/validator-request/) is to become your own operator.

Author: ```suggestion
- The [application process](https://sync.global/validator-request/) will remain in place for requests to become a Validator Operator.
```

## CIP 0050

<pre>
  CIP: CIP 0050
  Title: Controlled Validator Growth Through Trusted Operators
  Author: Wayne Collier 
  Status: Withdrawn
  Type: Process
  Created: 2025-03-11
  License: CC1
</pre>

## Abstract

> - Allow existing Validator operators to create up to 10 additional Validators for third-party participants.

Reviewer 3: What does "create" mean in this context? Does this mean operate a node or invite by providing a token?

- Allow existing Validator operators to create up to 10 additional Validators for third-party participants.
- Redirect new individual Validator applicants to approved existing operators.
## Specification

> - Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants.

Reviewer: ```suggestion
- Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants. Once an existing Validator operator has spun up the ten (10) allotted nodes, they can request an additional alottment from the Tokenomics committee. 
```

> -Whitelisting requests, if the node will not share an existing IP address, will come from the existing Validator operator who sponsors the new node

Reviewer 3: ```suggestion
- Whitelisting requests, if the node will not share an existing IP address, will come from the existing Validator operator who sponsors the new node
```

In the case that the onboarded node receives its own IP address in the whitelist, the whitelist will indicate the Validator Operator as the responsible party for the node. 

> -Credit for onboarding the Validator goes to the sponsoring Validator Operator

Reviewer 3: Does "onboarding" here has the same meaning as "create" above? If so should the same term be used? If not should this be clarified?

> -Credit for onboarding the Validator goes to the sponsoring Validator Operator

Reviewer 3: Does  "sponsoring" has the same meaning as "creating" and "onboarding" in the context of this CIP?

> -Credit for onboarding the Validator goes to the sponsoring Validator Operator

Reviewer 3: ```suggestion
- Credit for onboarding the Validator goes to the sponsoring Validator Operator
```

> -The Validator onboarding secret will be issued by a Super Validator, as before.

Reviewer 3: ```suggestion
- The Validator onboarding secret will be issued by a Super Validator, as before.
```

- Pre-approved operators, such as Kiln, Dfns and Copper, who have already been granted the right to operate up to 30 validator nodes, will retain that right.

> - Operators such as IEU, MPCH, Proof Group, and others already running their own Validators will be given discretion in onboarding third-party Validators.

Reviewer: ```suggestion
- Operators will be given discretion in onboarding third-party Validator nodes. After reaching the limit of ten additional nodes onboarded through this process, these operators may request additional node hosting permissions from the GSF Tokenomics committee. 
```

- This policy ensures a structured and scalable approach to Validator expansion while maintaining network integrity and supportability.

> - The [application process](https://sync.global/validator-request/) will remain in place for requests to become a Validator Operator.

Reviewer 3: no, this is me wrongly reading

  Author: <list of authors' real names>
* Discussions-To: <email address>
* Comments-Summary: <summary tone>
* Comments-URI: <links to wiki page for comments>
  Status: <Draft | Active | Proposed | Deferred | Rejected |
           Withdrawn | Final | Replaced | Obsolete>
  Type: <Standards Track | Governance | Tokenomics | Process | Informational>
  Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
  License: <abbreviation for approved license(s)>
* License-Code: <abbreviation for code under different approved license(s)>
* Post-History: <dates of postings to [https://lists.sync.global/g/cip-discuss] mailing list, or link to thread in mailing list archive>
* Requires: <CIP number(s)>
* Replaces: <CIP number>
* Superseded-By: <CIP number>
</pre>

## Abstract

- Allow existing Validator operators to create up to 10 additional Validators for third-party participants.
- Redirect new individual Validator applicants to approved existing operators.
- Maintain a controlled, trusted, and orderly network expansion.

## Specification

- Existing Validator operators will be authorized to create up to 10 additional Validators for third-party participants.

> - Third-party applicants wishing to run a Validator will be redirected to an existing operator.

Reviewer 4: Does this mean we are not accepting new applicants that want to run their own node anymore?

Reviewer 4: or does it just mean that you can get a node run by an existing validator operator without going through an approval process but the approval process is still there for applicants wanting to run their own node?

Author: 2nd case. The application is to become your own operator.

Reviewer 4: makes sense, consider folding this comment thread into the text. At least to me it's not clear just from the text that this is what is meant.

Author: Good suggestion. Added a bullet and link to the form
